broker/cloud_run/lsst/classify_upsilon/main.py

```diff
@@ -62,78 +60,82 @@ def run() -> tuple[str, int]:
     except pittgoogle.exceptions.BadRequest as exc:
         return str(exc), HTTP_400
 
-    upsilon_dict = _classify_with_upsilon(alert_lite)
-
-    TOPIC.publish(
-        pittgoogle.Alert.from_dict(
-            {"alert_lite": alert_lite.dict, "upsilon": upsilon_dict},
-            attributes={
-                **alert_lite.attributes,
-                "pg_upsilon_u_label": upsilon_dict["u_label"],
-                "pg_upsilon_u_flag": upsilon_dict["u_flag"],
-                "pg_upsilon_g_label": upsilon_dict["g_label"],
-                "pg_upsilon_g_flag": upsilon_dict["g_flag"],
-                "pg_upsilon_r_label": upsilon_dict["r_label"],
-                "pg_upsilon_r_flag": upsilon_dict["r_flag"],
-                "pg_upsilon_i_label": upsilon_dict["i_label"],
-                "pg_upsilon_i_flag": upsilon_dict["i_flag"],
-                "pg_upsilon_z_label": upsilon_dict["z_label"],
-                "pg_upsilon_z_flag": upsilon_dict["z_flag"],
-                "pg_upsilon_y_label": upsilon_dict["y_label"],
-                "pg_upsilon_y_flag": upsilon_dict["y_flag"],
-            },
-            schema_name="default",
-        )
+    # UPSILoN recommends using light curves with more than 80 data points
+    has_min_detections_in_any_band = any(
+        alert_lite.dict["variability"].get(f"n_detections_{band}_band", 0) >= 80
+        for band in survey_bands
     )
 
-    return "", HTTP_204
+    if has_min_detections_in_any_band:
+        alert_lite_df = _create_lite_dataframe(alert_lite.dict["alert_lite"])
+        upsilon_dict = _classify_with_upsilon(alert_lite_df)
+        TOPIC.publish(
+            pittgoogle.Alert.from_dict(
+                {
+                    "alert_lite": alert_lite.dict,
+                    "upsilon": {
+                        "diaObjectId": alert_lite.dict["diaObject"]["diaObjectId"],
+                        "diaSourceId": alert_lite.dict["diaSource"]["diaSourceId"],
+                        **upsilon_dict,
+                    },
```

Comment:
Just realized, if this module is triggered by the 'variability' stream, should there be another key in here passing that dict on?